src/engine.jl

```diff
@@ -59,7 +59,7 @@ function close(session::MSession)
     # Close a MATLAB Engine session
     @assert libeng::Ptr{Void} != C_NULL
     ret = ccall(engfunc(:engClose), Cint, (Ptr{Void},), session)
-    ret != 0 && throw(MEngineError("Failed to close a MATLAB engine session (err = $r)"))
+    ret != 0 && throw(MEngineError("failed to close a MATLAB engine session (err = $ret)"))
```

Comment:
`engfunc(:engSetVisible )` below. Apparently @github puts the comment on the wrong line if it was made with whitespace changes removed. You can see it at the right line with https://github.com/JuliaInterop/MATLAB.jl/pull/79/files?w=0 .